src/OpenDebugAD7/AD7DebugSession.cs

```diff
@@ -1193,6 +1197,87 @@ protected override void HandlePauseRequestAsync(IRequestResponder<PauseArguments
             m_program.CauseBreak();
             responder.SetResponse(new PauseResponse());
         }
+        
+        protected override void HandleGotoRequestAsync(IRequestResponder<GotoArguments> responder)
+        {
+            var response = new GotoResponse();
```

Comment:
>var [](start = 12, length = 3)

@WardenGnaw would know better than I, but following the pattern, I think you want to wrap these in a `try`+`catch` where the catch will do a `responder.SetError(new ProtocolException(e.Message));`

Reply:
Yes there are different patterns in the code.
Also something like extract inner exception etc.